base/task.jl

```diff
@@ -504,19 +469,6 @@ function schedule(t::Task, @nospecialize(arg); error=false)
     return t
 end
 
-# fast version of `schedule(t, arg); wait()`
-function schedule_and_wait(t::Task, @nospecialize(arg)=nothing)
-    (t.state == :runnable && t.queue === nothing) || error("schedule: Task not runnable")
-    W = Workqueues[Threads.threadid()]
-    if isempty(W)
-        return yieldto(t, arg)
-    else
-        t.result = arg
-        push!(W, t)
-    end
-    return wait()
-end
```

Comment:
hm, curious, so no users of this left? I guess that's a good sign—although I hope someone wasn't too dependent on whatever "fast version" meant